frontend/packages/topology/integration-tests/support/pages/topology/topology-page.ts

```diff
@@ -168,9 +173,32 @@ export const topologyPage = {
   rightClickOnNode: (releaseName: string) => {
     topologyPage.getNode(releaseName).trigger('contextmenu', { force: true });
   },
+  rightClickOnApplicationGroupings: (appName: string) => {
+    const id = `[data-id="group:${appName}"]`;
+    cy.get(id)
+      .should('be.visible')
+      .first()
+      .trigger('contextmenu', { force: true });
+  },
   clickOnNode: (releaseName: string) => {
     topologyPage.getNode(releaseName).click({ force: true });
   },
+  clickOnApplicationGroupings: (appName: string) => {
+    const id = `[data-id="group:${appName}"]`;
+    cy.get(id)
+      .should('be.visible')
+      .first()
+      .click({ force: true });
+  },
+  verifyApplicationGroupingsDeleted: (appName: string) => {
+    const id = `[data-id="group:${appName}"]`;
+    cy.get(id, { timeout: 50000 }).should('not.exist');
+  },
+  verifyApplicationGroupings: (workloadName: string) => {
+    cy.get(topologyPO.sidePane.applicationGroupingsTitle).should('be.visible');
+    const id = `[data-test-id="${workloadName}"]`;
+    cy.get(id).should('be.visible');
```

Comment:
We can club this together as:
```suggestion
    cy.byTestID(workloadName)
```

Reply:
Updated